diff -burN centericq-4.9.6-orig/kkconsui-0.1/Makefile.am centericq-4.9.6-am/kkconsui-0.1/Makefile.am
--- centericq-4.9.6-orig/kkconsui-0.1/Makefile.am	2003-08-21 23:24:32.000000000 +0200
+++ centericq-4.9.6-am/kkconsui-0.1/Makefile.am	2003-09-02 10:14:57.928695392 +0200
@@ -5,8 +5,3 @@
 SUBDIRS = include src
 EXTRA_DIST = config.guess config.sub
 AUTOMAKE_OPTIONS = foreign
-libkkconsui_a_LIBADD = src/treeview.o src/colorschemer.o src/textinputline.o src/fileselector.o src/textwindow.o src/texteditor.o src/dialogbox.o src/screenarea.o src/textbrowser.o src/abstractui.o src/horizontalbar.o src/linkedlist.o src/cmenus.o src/conscommon.o
-libkkconsui_a_LIBADD = src/treeview.o src/colorschemer.o src/textinputline.o src/fileselector.o src/textwindow.o src/texteditor.o src/dialogbox.o src/screenarea.o src/textbrowser.o src/abstractui.o src/horizontalbar.o src/linkedlist.o src/cmenus.o src/conscommon.o
-libkkconsui_a_LIBADD = src/texteditor.o src/horizontalbar.o src/textinputline.o src/abstractui.o src/textbrowser.o src/treeview.o src/textwindow.o src/dialogbox.o src/cmenus.o src/linkedlist.o src/conscommon.o src/screenarea.o src/fileselector.o src/colorschemer.o
-libkkconsui_a_LIBADD = src/texteditor.o src/horizontalbar.o src/textinputline.o src/abstractui.o src/textbrowser.o src/treeview.o src/textwindow.o src/dialogbox.o src/cmenus.o src/linkedlist.o src/conscommon.o src/screenarea.o src/fileselector.o src/colorschemer.o
-libkkconsui_a_LIBADD = src/texteditor.o src/horizontalbar.o src/textinputline.o src/abstractui.o src/textbrowser.o src/treeview.o src/textwindow.o src/dialogbox.o src/cmenus.o src/linkedlist.o src/conscommon.o src/screenarea.o src/fileselector.o src/colorschemer.o
diff -burN centericq-4.9.6-orig/libicq2000-0.1/Makefile.am centericq-4.9.6-am/libicq2000-0.1/Makefile.am
--- centericq-4.9.6-orig/libicq2000-0.1/Makefile.am	2003-08-21 23:24:33.000000000 +0200
+++ centericq-4.9.6-am/libicq2000-0.1/Makefile.am	2003-09-02 10:15:25.002579536 +0200
@@ -4,8 +4,3 @@
 SUBDIRS = libicq2000 src
 EXTRA_DIST = stamp-h.in
 AUTOMAKE_OPTIONS = foreign
-liblibicq2000_a_LIBADD = src/buffer.o src/Client.o src/Contact.o src/ContactList.o src/DirectClient.o src/events.o src/exceptions.o src/ICBMCookie.o src/ICQ.o src/SNAC-base.o src/SNAC-BUD.o src/SNAC-GEN.o src/SNAC-LOC.o src/SNAC-MSG.o src/SNAC-SRV.o src/SNAC-UIN.o src/SNAC.o src/socket.o src/TLV.o src/Translator.o src/UserInfoBlock.o src/Xml.o src/SNAC-BOS.o src/SNAC-SBL.o src/version.o src/userinfohelpers.o src/SMTPClient.o src/SocketClient.o src/time_extra.o src/MessageHandler.o src/Capabilities.o src/ContactTree.o
-liblibicq2000_a_LIBADD = src/Client.o src/Contact.o src/ContactList.o src/DirectClient.o src/ICBMCookie.o src/ICQ.o src/SNAC-BOS.o src/SNAC-BUD.o src/SNAC-GEN.o src/SNAC-LOC.o src/SNAC-MSG.o src/SNAC-SBL.o src/SNAC-SRV.o src/SNAC-UIN.o src/SNAC-base.o src/SNAC.o src/TLV.o src/Translator.o src/UserInfoBlock.o src/Xml.o src/buffer.o src/events.o src/exceptions.o src/socket.o src/userinfohelpers.o src/version.o src/SMTPClient.o src/SocketClient.o src/time_extra.o src/Capabilities.o src/MessageHandler.o src/ContactTree.o
-liblibicq2000_a_LIBADD = src/Client.o src/Contact.o src/ContactList.o src/DirectClient.o src/ICBMCookie.o src/ICQ.o src/SNAC-BOS.o src/SNAC-BUD.o src/SNAC-GEN.o src/SNAC-LOC.o src/SNAC-MSG.o src/SNAC-SBL.o src/SNAC-SRV.o src/SNAC-UIN.o src/SNAC-base.o src/SNAC.o src/TLV.o src/Translator.o src/UserInfoBlock.o src/Xml.o src/buffer.o src/events.o src/exceptions.o src/socket.o src/userinfohelpers.o src/version.o src/SMTPClient.o src/SocketClient.o src/time_extra.o src/Capabilities.o src/MessageHandler.o src/ContactTree.o
-liblibicq2000_a_LIBADD = src/buffer.o src/Client.o src/Contact.o src/ContactList.o src/DirectClient.o src/events.o src/exceptions.o src/ICBMCookie.o src/ICQ.o src/SNAC-base.o src/SNAC-BUD.o src/SNAC-GEN.o src/SNAC-LOC.o src/SNAC-MSG.o src/SNAC-SRV.o src/SNAC-UIN.o src/SNAC.o src/socket.o src/TLV.o src/Translator.o src/UserInfoBlock.o src/Xml.o src/SNAC-BOS.o src/SNAC-SBL.o src/version.o src/userinfohelpers.o src/SMTPClient.o src/SocketClient.o src/time_extra.o src/MessageHandler.o src/Capabilities.o src/ContactTree.o
-liblibicq2000_a_LIBADD = src/buffer.o src/Client.o src/Contact.o src/ContactList.o src/DirectClient.o src/events.o src/exceptions.o src/ICBMCookie.o src/ICQ.o src/SNAC-base.o src/SNAC-BUD.o src/SNAC-GEN.o src/SNAC-LOC.o src/SNAC-MSG.o src/SNAC-SRV.o src/SNAC-UIN.o src/SNAC.o src/socket.o src/TLV.o src/Translator.o src/UserInfoBlock.o src/Xml.o src/SNAC-BOS.o src/SNAC-SBL.o src/version.o src/userinfohelpers.o src/SMTPClient.o src/SocketClient.o src/time_extra.o src/MessageHandler.o src/Capabilities.o
