--- configure.in~	Mon Feb  9 20:14:30 2004
+++ configure.in	Mon Feb  9 20:17:55 2004
@@ -92,7 +92,7 @@
 
 AC_CHECK_HEADERS([hash_map strstream ext/hash_map sstream],,)
 
-AM_ICONV_LINK
+# AM_ICONV_LINK
 
 AC_CHECK_LIB(socket, socket)
 AC_CHECK_LIB(nsl, gethostbyname)
--- kkstrtext-0.1/configure.in~	Fri Jan 16 01:21:49 2004
+++ kkstrtext-0.1/configure.in	Mon Feb  9 20:21:33 2004
@@ -10,7 +10,7 @@
 AC_CHECK_HEADERS(_G_config.h)
 AC_LANG_CPLUSPLUS
 
-AM_ICONV_LINK
+#AM_ICONV_LINK
 AM_ICONV
 
 CFLAGS="$CFLAGS $INCICONV"
