diff -urN centericq-4.9.11.org/configure.in centericq-4.9.11/configure.in
--- centericq-4.9.11.org/configure.in	2004-03-23 03:23:14.247557344 +0100
+++ centericq-4.9.11/configure.in	2004-03-23 03:24:55.288196832 +0100
@@ -84,13 +84,6 @@
     else
 	AC_MSG_RESULT([not found or disabled])
 	with_ssl="no"
-
-	AM_PATH_LIBGNUTLS_EXTRA(0.0.1, [
-	    CXXFLAGS="$CXXFLAGS $LIBGNUTLS_CFLAGS"
-	    LIBS="$LIBS $LIBGNUTLS_LIBS -lgnutls-extra"
-	    AC_DEFINE(HAVE_GNUTLS, 1, [use gnutls])
-	    with_ssl="yes"
-	])
     fi
 fi
 
diff -urN centericq-4.9.11.org/connwrap-0.1/configure.in centericq-4.9.11/connwrap-0.1/configure.in
--- centericq-4.9.11.org/connwrap-0.1/configure.in	2004-03-23 03:23:16.856160776 +0100
+++ centericq-4.9.11/connwrap-0.1/configure.in	2004-03-23 03:24:39.997521368 +0100
@@ -44,12 +44,6 @@
 	AC_CHECK_LIB(ssl, main)
     else
 	AC_MSG_RESULT([not found or disabled])
-
-	AM_PATH_LIBGNUTLS_EXTRA(0.0.1, [
-	    CXXFLAGS="$CXXFLAGS $LIBGNUTLS_CFLAGS"
-	    LIBS="$LIBS $LIBGNUTLS_LIBS -lgnutls-extra"
-	    AC_DEFINE(HAVE_GNUTLS)
-	])
     fi
 fi
 
